cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/SepaDataDTO.java

```diff
@@ -4,7 +4,7 @@
 import org.hibernate.validator.constraints.NotEmpty;
 
 /**
- * @deprecated by {@link TokenDataDTO}
+ * @deprecated by {@link StripeTokenDTO}
  */
 @Deprecated
 public class SepaDataDTO extends PaymentDataDTO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)